packages/build-rpm-packages.sh

```diff
@@ -160,18 +160,22 @@ if [ -f "/etc/redhat-release" ]; then
     release_version=".el$(cat /etc/redhat-release | grep -o -E '[0-9]+' | head -1)"
 elif [ -f "/etc/centos-release" ]; then
     release_version=".el$(cat /etc/centos-release | grep -o -E '[0-9]+' | head -1)"
+elif [ -f "/etc/os-release" ]; then
+    . /etc/os-release
```

Comment:
It is not a good idea to `source /etc/os-release` here without protection. It could lead to conflicting variables later. For example, what if we use `$ID` or `$VERSION_ID` in the future? Are there other variables that /etc/os-release might replace?

Reply:
I will submit a patch with more safeguards.